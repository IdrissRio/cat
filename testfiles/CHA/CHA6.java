interface Box<T> {

}

class MyBox<T> implements Box<T> {

}