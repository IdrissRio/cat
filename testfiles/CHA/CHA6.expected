Box :> [MyBox]
Box <: []
MyBox :> []
MyBox <: [Box]